factory/network/networkComponents.go

```diff
@@ -253,6 +234,86 @@ func (ncf *networkComponentsFactory) createPeerHonestyHandler(
 	return peerHonesty.NewP2pPeerHonesty(ratingConfig.PeerHonesty, pkTimeCache, suCache)
 }
 
+func (ncf *networkComponentsFactory) createNetworkHolder(
+	peersHolder p2p.PreferredPeersHolderHandler,
+	p2pConfig p2pConfig.P2PConfig,
+	logger p2p.Logger,
+) (networkComponentsHolder, error) {
+
```

Comment:
remove this empty line?

Reply:
it is more readable this way, with the params split on multiple lines